apps/nextra/components/landing/sections/TestimonialsSection.tsx

```diff
@@ -13,36 +13,38 @@ interface Testimonial {
   authorLogo: ReactNode;
 }
 
+interface PlaceholderAuthorLogoProps {
+  src: string;
+}
+
+const PlaceholderAuthorLogo = ({ src }: PlaceholderAuthorLogoProps) => (
+  <Image src={src} alt="placeholder logo" height={42} width={42} />
+);
+
 export function TestimonialsSection() {
   const { locale } = useRouter();
   const t = i18nConfig[locale!];
 
   // TODO: Each testimonial should have its own logo based on the organization that the author is from
-  const placeholderAuthorLogo = (
-    <Image
-      src="/landing/testimonial-logo.png"
-      alt="placeholder logo"
-      height={42}
-      width={42}
-    />
-  );
 
   const testimonials: Testimonial[] = useMemo(
     () => [
       {
         body: t.testimonial1Body,
         author: t.testimonial1Author,
-        authorLogo: placeholderAuthorLogo,
+        authorLogo: PlaceholderAuthorLogo({ src: "/landing/zabava.png" }),
```

Comment:
This should be invoked with JSX: `<PlaceholderAuthorLogo src="/landing/zabava.png" />`
Same with the two below as well.